tests/models/test_taskinstance.py

```diff
@@ -846,9 +846,20 @@ def test_log_url(self):
             'execution_date=2018-01-01T00%3A00%3A00%2B00%3A00'
             '&task_id=op'
             '&dag_id=dag'
+            '&try_number=1'
```

Comment:
Good to have a case in which we don't provide `try_number`.

Reply:
this is expected url, it can't be generated without try_number parameter since every TaskInstance has try_number